css/main.css

```diff
@@ -1,20 +1,37 @@
 body {
     margin: 60px auto;
-    width: 70%;
+    width: 80%;
+	background-color: #617C8F;
+	padding-top: 40px;
+}
+header {
+	padding: 0.2em;
+    color: white;
+    background-color: #000000;
+    clear: left;
+    text-align: left;
+	font-size: 3em;
+	width: 100%;
+	position: fixed;
+	top: 0px;
+	left: 0px;
+	border-bottom: 3px solid white;
+	
 }
 nav ul, footer ul {
-    font-family:'Helvetica', 'Arial', 'Sans-Serif';
-    padding: 0px;
+    font-family:'Helvetica';
```

Comment:
Was there a reason for removing the font hierarchy? It might be better to put Arial and Sans-Serif back in for computers that don't have Helvetica, otherwise we get stuck with TNR